tests/utils.py

```diff
@@ -55,7 +56,8 @@ def return_to_saved_path():
 
 @pytest.fixture(scope='function')
 def test_config(testing_workdir, request):
-    return Config(croot=testing_workdir, anaconda_upload=False, verbose=True, activate=False)
+    return Config(croot=testing_workdir, anaconda_upload=False, verbose=True, activate=False,
+                  noverify=False)
```

Comment:
why is it necessary to pass noverify here?  For the test configuration, things should be mostly defaults.  The stuff I have overridden here is just some tweaks to make the test stuff not muck up the system they're running on.